index.html

```diff
@@ -10,7 +10,57 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <main>
+        <header>
+            <h1>Weather Report</h1>
+        </header>
+    
+        <h3>For the lovely city of Seattle</h3>
+        <div id="temp-city">
+        <section class="temp_section">
+                <section id="temp_display">Temperature</section>
+                    <div class="temp-change-buttons_div"> 
+                        <button id="up-arrow" type="button">⬆️ </button>
+                        <div id="temp-count">0</div>
+                        <button id="down-arrow" type="button">⬇️</button>
+                    </div>
+                </section>
+                <section class="city-query">
+                    <label for="city-name">🏙️ City Name 🏙️</label>
+                    <input type="text" id="city-name" size="20">
+                    <span>
+                        <button id="realtime-button-weather" type="button" alt="Get Realtime Temperature">Get Realtime Temperature</button>
+                        <button id="reset-city-name" alt="Reset Temperature">Reset</button></span>
+                    </input>
+                </section>
+        </section>
+        </div>
+
+        <section class="garden-section">
+        <h2>Weather Garden</h2>
+            <div id="landscape_display" class="garden_content">
+                <div id="sky">🌞</div>
+                <div id="landscape">🌵🌼🌸🍁🌵🌼🌸🍁</div>
+            </div>
+        </section>
+
+            <section class="sky-section">
+                <label for="sky-select">Sky
+                    <select class="sky-select" name="sky-select">
+                        <option value="">--Please choose an option--</option>
+                        <option value="sunny">Sunny</option>
+                        <option value="cloudy">Cloudy</option>
+                        <option value="rainy">Rainy</option>
+                        <option value="snowy">Snowy</option>
+                    </select>
```

Comment:
[Great job remembering to include a `value` field to the `option` HTML tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select)!